### PR TITLE
Labels layer tutorial: added notes on how to split,merge,delete,draw objects using tools

### PR DESCRIPTION
**Pull request by VolkerH**
This pull request makes some changes to the labels layer tutorial.

When I think about label images, I think of objects (connected components) and operations like creating, splitting, merging and deleting objects. This is a different way of thinking than paintbrush and fill bucket concepts which are concepts from drawing/graphics programs. While the tutorial already gave some examples on how to use these tools to edit objects, I explicitly show how to perform the object editing operations mentioned above with the given tools. Each operation is illustrated with a screen recording (animated gif).

Weaknesses: I am not addressing 3D an non-contiguous objects in these changes.

### CODE REVIEW

**Comment by VolkerH** [comment]
I should also add that I can't preview the pages in my markdown editor with images embedded using the way you include pictures for the tutorials. I assume there is some magic happening during CI when the website is generated.

**Comment by VolkerH** [comment]
@jni @kne42, comments ?

File: fundamentals/labels.md [hunk 1]
```diff
@@ -126,19 +127,43 @@ If you have a multidimensional labels layer then your paintbrush will only edit
...
+
+Using the `color picker`, `paintbrush` and `fill bucket` tools one can create and edit object segmentation maps. Below we show how to use these tools to by perform common editing tasks on connected components (keep the `contiguous` box checked).
+
+**drawing a connected component**:
+
+![image]({{ '/assets/tutorials/draw_component.gif' | relative_url }})
+
+Press `M` to select a new label color. Select the `paintbrush` tool and draw a closed contour around the object. Select the `fill bucket` tool and click inside the contour to assign the label to all pixels of the object.
+
+**seleting a connected component**:
```
Comment (jni) [inline fundamentals/labels.md]:
```suggestion
**selecting a connected component**:
```

**Comment by VolkerH** [comment]
thanks and no worries. not surprised that things get buried in the pile at the rate that things are moving with napari, therefore the bump.

**Comment by VolkerH** [comment]
Does this need another review @jni, @kne42, @sofroniewn   ?

**Comment by VolkerH** [comment]
Thanks !

**Pull request merged**